documentation/staging/content/quickstart/install.md

```diff
@@ -2,69 +2,74 @@
-    --set "kubernetes.namespaces={traefik}"
-```
-
-#### Install the operator.
-
-1.  Create a namespace for the operator:
+1. Create a namespace for the operator:
 
     ```shell
     $ kubectl create namespace sample-weblogic-operator-ns
     ```
 
-2.	Create a service account for the operator in the operator's namespace:
+1. Create a service account for the operator in the operator's namespace:
 
     ```shell
     $ kubectl create serviceaccount -n sample-weblogic-operator-ns sample-weblogic-operator-sa
     ```
 
-3.  Use `helm` to install and start the operator from the directory you just cloned:	 
+1. Set up Helm with the location of the operator Helm chart using this format: `helm repo add <helm-chart-repo-name> <helm-chart-repo-url>`.
 
-    ```shell
-    $ helm install sample-weblogic-operator kubernetes/charts/weblogic-operator \
-      --namespace sample-weblogic-operator-ns \
-      --set serviceAccount=sample-weblogic-operator-sa \
-      --wait
     ```
+    $ helm repo add weblogic-operator https://oracle.github.io/weblogic-kubernetes-operator/charts --force-update  
+    ```
+ 1. Install the operator using this format: `helm install <helm-release-name> <helm-chart-repo-name>/weblogic-operator ...`
+
+     ```shell
+     $ helm install sample-weblogic-operator weblogic-operator/weblogic-operator \
+       --namespace sample-weblogic-operator-ns \
+       --set serviceAccount=sample-weblogic-operator-sa \
+       --wait
+     ```
+     This Helm release deploys the operator and configures it with the default behavior to manage Domains in any Kubernetes namespace with the label, `weblogic-operator=enabled`.
+
+#### Create a Traefik ingress controller.
+
+1. Set up Helm with the location of the Traefik Helm chart using this format: `helm repo add <helm-chart-repo-name> <helm-chart-repo-url>`.
+
+   ```shell
+   $ helm repo add traefik https://helm.traefik.io/traefik --force-update
+   ```
+
+1. Create a namespace for the ingress controller.
+
+   ```shell
+   $ kubectl create namespace traefik
+   ```
 
-    This Helm release deploys the operator and configures it with the default behavior to manage Domains in any Kubernetes namespace with the label, `weblogic-operator=enabled`.
+1. Install Traefik using this format: `helm install <helm-release-name> <helm-chart-repo-name>/traefik ...`
 
+   ```shell
+   $ helm install traefik-operator traefik/traefik \
+       --namespace traefik \
+       --set "ports.web.nodePort=30305" \
+       --set "ports.websecure.nodePort=30443" \
+       --set "kubernetes.namespaces={traefik}"
+   ```
+    This deploys the Traefik controller with plain text node port `30305`, SSL node port `30443`, and `kubernetes.namespaces` specifically set.
 
-4. Verify that the operator's pod is running, by listing the pods in the operator's namespace. You should see one
-for the operator.
+1. Verify that the operator's pod is running by listing the pods in the operator's namespace. You should see one
```

Comment:
This step and the following belong under the 'operator steps', not the 'traefik steps'. They were not moved when we put the operator as the first action.